index.py

```diff
@@ -10,10 +10,11 @@
 from rnaseqc.over_time import layout as rnaseqc_overtime_layout
 from runreport.proj_hist import layout as runreport_projhist_layout
 from runscanner.yield_over_time import layout as runscanner_yield_over_time_layout
+from poolingQC_sample import layout as poolingQC_layout
```

Comment:
Could `poolingQC_sample` please be converted to `pooling_qc_sample` to match the snake case throughout the rest of the project?